### DISCUSSION TITLE
Still supported?

### DISCUSSION THREAD

**Discussion by jerone** [papaia/prettignore#28] [Q&A] [answered]
This is the first and only package I found that supports formatting ignore files.

A few questions:

1. Is this package still maintained?
2. Thought of making this an [Prettier plugin](https://prettier.io/docs/en/plugins.html)?
3. Thought of making this an [ESLint rule](https://eslint.org/docs/latest/developer-guide/working-with-rules)/[plugin](https://eslint.org/docs/latest/developer-guide/working-with-plugins)/[parser](https://eslint.org/docs/latest/developer-guide/working-with-custom-parsers)?

**Comment by papaia** [comment, marked as answer]
1. Not actively at the moment. This was created just as an experiment for my projects and once it did what I wanted it to do I pretty much left it untouched. If you're interested in any changes, I'd be happy to respond, but, I can't guarantee any major further work.
2. Prettier uses actual parsers for its formatting while Prettignore uses _very_ simple string manipulations, so that would be overkill (or simply too much work for not much added value, in my opinion).
3. No, however, it is very similar to the Prettier case above.

**Reply by jerone** [reply]
Valid answers. Thank you.
